src/useDynamicAttribute/index.stories.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import { FixedSizeList } from 'react-window'
+
+import { storiesOf } from '@storybook/react'
+import { action } from '@storybook/addon-actions'
+
+import { useDynamicAttribute } from '.'
+import { FLAT_LIST_TEST_DATA } from './__stories__/seed'
+
+const CheckableList = () => {
+  const data = useDynamicAttribute(FLAT_LIST_TEST_DATA, 'checked', 'setChecked', (item) => item.checked) 
+
+  return (
+    <FixedSizeList height={600} width={400} itemSize={50} itemData={data} itemCount={data.length}>
+      {Item}
+    </FixedSizeList>
+  )
+}
+
+export default storiesOf('List hooks', module)
+  .add('useDynamicAttribute', () => <CheckableList />)
+
+const Item = ({ data, index, style }) => {
+  const item = data[index]
+  return (
+    <div key={index} style={style}>
+    {item.value}: 
+      <input type='checkbox' checked={item.checked} onChange={(e) => {
+        item.setChecked(e.target.checked, data)
```

Comment:
That does not work as expected because it will reset every item if another item changes. I will look into it to see if I can make it work another way.

Reply:
yeah, useExpandable is alrdy doing it -> its a bit tricky as you have to play around with references.
The goal is that objects which have been altered, have a new reference. So item changed means new item in an new array.